fine-tuning/models.tsp

```diff
@@ -162,6 +221,45 @@ model FineTuningJob {
 
   /** The file ID used for validation. You can retrieve the validation results with the [Files API](/docs/api-reference/files/retrieve-contents). */
   validation_file: string | null;
+
+  /** A list of integrations to enable for this fine-tuning job. */
+  integrations?: FineTuningJobIntegrations | null;
+
+  /** The seed used for the fine-tuning job. */
+  seed: int32;
+
+  /** The Unix timestamp (in seconds) for when the fine-tuning job is estimated to finish. The value will be null if the fine-tuning job is not running. */
+  estimated_finish?: int32 | null;
```

Comment:
Should this be a `duration` with the `@encode` decorator?

Reply:
sure looks like it! adding another regex rule for `.*estimated_finish`.